src/content/structured/get-started/gatsby.mdx

```diff
@@ -0,0 +1,79 @@
+---
+path: "/get-started/install-components/gatsby"
+
+navPriority: 7
+
+date: "2023-01-19"
+
+title: "Gatsby"
+
+subTitle: "How to use the components in a Gatsby project."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/gatsby.mdx"
+---
+
+## Step one
+
+In the root of your project:
+
+```shell
+// using npm
+npm install @ukic/react @ukic/fonts
+
+// using yarn
+rm package-lock.json
+yarn add @ukic/react @ukic/fonts
+```
+
+## Step Two
+
+You need to intstall an extra package to allow Gatsby to render the components server side.
+
+```shell
+// using npm
+npm install --save gatsby-plugin-stencil
+```
+
+## Step three
+
+In your `gatsby-config.js` file (at the root of the project) add the following to the `module.exports`:
+
+```js
+plugins: [
+  {
+    resolve: `gatsby-plugin-stencil`,
+
+    options: {
+      module: "@ukic/web-components",
+      renderToStringOptions: {
+        prettyHtml: true,
+      },
+    },
+  },
+];
+```
+
+## Step four
+
+Import the component(s) in your Gatsby pages.
+
+```jsx
+import { IcComponent } from "@ukic/react";
+```
+
+## Step five
+
+To get the correct styling with the ICDS components, import the core CSS file.
+
+Add the following to the `index.jsx` file of your project.
+
+```jsx
+project / src / pages / index.jsx;
```

Comment:
It's possible the developer is using the TS template for Gatsby perhaps say something 

`Add the following to the top level index file for your project` and remove the folder destination from the code example.